src/components/exchange/ExchangeHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { SheetHandle } from '../sheet';
+import { Heading, Inset, Stack } from '@/design-system';
+
+interface ExchangeHeaderProps {
+  testID: string;
+  title: string;
+}
+
+export default function ExchangeHeader({ testID, title }: ExchangeHeaderProps) {
+  return (
+    <Inset space="6px">
```

Comment:
not having a parent <Box testID={'something'} >...<Box> is breaking e2e 

![image](https://user-images.githubusercontent.com/12115171/188028035-f52332bf-e90a-4529-b7ff-f330c65ad060.png)

before the container had the testID param, now the testID param is dependent on `title` being defined